templates/centos6.tpl.yaml

```diff
@@ -66,16 +66,16 @@ objects:
       metadata:
         name: ${NAME}
       spec:
-        pvc:
+        storage:
+          dataSource:
```

Comment:
This should be a source at DataVolume level as it was previously. To correctly use CDI import logic. 

By defining dataSource here, you are using the `pvc.dataSource` which is a kubernetes extension (https://docs.openshift.com/container-platform/4.6/storage/container_storage_interface/persistent-storage-csi-cloning.html) that only supports some storage providers.

Reply:
you are right. I was mistify by one of our script, which modified wrong value. That was why it showed the error messages